checkov/main.py

```diff
@@ -104,8 +106,8 @@ def run(banner=checkov_banner, argv=sys.argv[1:]):
                                                         skip_fixes=config.skip_fixes,
                                                         skip_suppressions=config.skip_suppressions,
                                                         source=source, source_version=source_version, repo_branch=config.branch)
-            excluded_paths = bc_integration.get_excluded_paths()
-            runner_filter.excluded_paths = excluded_paths
+            platform_excluded_paths = bc_integration.get_excluded_paths()
+            runner_filter.excluded_paths = (runner_filter.excluded_paths + platform_excluded_paths) if runner_filter.excluded_paths else platform_excluded_paths
```

Comment:
@mikeurbanski1 Isn't `runner_filter.excluded_paths + platform_excluded_paths` enough? if `runner_filter.excluded_paths` is empty it should have the same effect as just using platform_excluded_paths IMO

Reply:
I wanted to guard against it somehow being set to `None` in the future, but you are right that it would work currently. I'll change it.

Reply:
@rotemavni Nope
```sh
Python 3.7.10 (default, Mar  3 2021, 14:09:10) 
[Clang 12.0.0 (clang-1200.0.32.29)] on darwin
None + ['a']
Traceback (most recent call last):
  File "<input>", line 1, in <module>
TypeError: unsupported operand type(s) for +: 'NoneType' and 'list'
```